ui/packages/app/src/components/Chatbot/Chatbot.svelte

```diff
@@ -30,7 +30,6 @@
 		/>
 	{/if}
 	<ChatBot
-		{style}
```

Comment:
we shouldn't get rid of this. I actually have a pr using style so devs can set the height of the chatbot (https://github.com/gradio-app/gradio/pull/3369)